pages/common/clamscan.md

```diff
@@ -0,0 +1,36 @@
+# clamscan
+
+> A command line virus scanner.
+> Homepage: <https://www.clamav.net>.
+
+- Scan a file for vulnerabilities:
+
+`clamscan {{path/to/file}}`
+
+- Scan all files recursively in a specific directory:
+
+`clamscan -r {{path/to/directory}}`
+
+- Scan data from stdin:
+
+`{{command}} | clamscan -`
+
+- Specify a virus database file or directory of files:
+
+`clamscan --database {{path/to/database_file_or_directory}}`
+
+- Output only infected files:
+
+`clamscan --infected`
```

Comment:
What does this scan?

Reply:
By default it scans the current directory if no file/directory are provided.

Reply:
Ah, I see. Maybe we could work that into this example somehow then? It doesn't seem obvious to me.

Reply:
Perhaps something like:
> - Scan the current directory and output only infected files:

Reply:
Sounds good!

Reply:
Sorry, been busy this weekend. Should be able to update this in a couple of hours. :+1:

Reply:
Updated it @sbrl